Cartfile

```diff
@@ -1,2 +1,2 @@
 github "lyft/Kronos" ~> 4.2
-github "microsoft/plcrashreporter" == 1.8.1
+github "microsoft/plcrashreporter" == 1.10.0
```

Comment:
this can/should be `~> 1.10.0`.
it was `== 1.8.1` just to avoid the problematic `1.9` version.

Reply:
Indeed - fixed. Now it's consistent with SPM and CP 💪